File: src/main/java/managers/BeautyManager.java
package managers;

import client.player.Beauty;

import java.util.LinkedHashMap;
import java.util.Map;

public class BeautyManager {

    private final static Map<Integer, Beauty> hairs = new LinkedHashMap<>();
    private final static Map<Integer, Beauty> faces = new LinkedHashMap<>();

    public static Map<Integer, Beauty> getHairs() {
        return hairs;
    }

    public static Map<Integer, Beauty> getFaces() {
        return faces;
    }
}

File: src/main/java/managers/BeautyManager.kt
package managers

import client.player.Beauty

object BeautyManager {

    val hairs: MutableMap<Int, MutableList<Beauty>> =
        LinkedHashMap<Int, MutableList<Beauty>>().also { map -> repeat(101) { map[it] = ArrayList() } }
    val faces: MutableMap<Int, MutableList<Beauty>> =
        LinkedHashMap<Int, MutableList<Beauty>>().also { map -> repeat(101) { map[it] = ArrayList() } }
}